src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -204,6 +209,86 @@ class SpanDetail extends React.Component<Props, State> {
     );
   }
 
+  renderSpanErrorMessage() {
+    const {orgId, spanErrors, totalNumberOfErrors, span, trace, eventView} = this.props;
+
+    if (spanErrors.length === 0 || totalNumberOfErrors === 0 || isGapSpan(span)) {
+      return null;
+    }
+
+    // invariant: spanErrors.length <= totalNumberOfErrors
```

Comment:
Instead of a comment should this be a condition that returns null if violated?

Reply:
The condition is there to ensure the presented message is correct. If the condition is violated, it's still useful to inform the user that the transaction has related errors.